js/views/QuickAddLegacy.svelte

```diff
@@ -11,12 +11,15 @@
 	import TimeEditor from "./TimeEditor.svelte";
 	import { onMount } from "svelte";
 	import { translate } from "@nextcloud/l10n";
+	import { Helpers } from "../lib/helpers";
 
 	$: show = false;
 	$: loading = false;
 	$: taskError = false;
 
 	let duration = 1;
+	let startTime = "00:00";
```

Comment:
Do you think it could be beneficial to set this to the current time or something closer to the current time? Midnight is probably a less frequent time for entries to start at 😅

Reply:
Changed to current time (`new Date()`).